src/Controls/samples/Controls.Sample.Profiling/Maui.Controls.Sample.Profiling.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-android;net6.0-maccatalyst;net6.0-ios</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+    <SingleProject>true</SingleProject>
+    <ApplicationTitle>Profiling</ApplicationTitle>
+    <ApplicationId>com.microsoft.maui.profiling</ApplicationId>
+    <ApplicationVersion>1.0</ApplicationVersion>
+    <AndroidVersionCode>1</AndroidVersionCode>
+    <IsPackable>false</IsPackable>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <RuntimeIdentifier Condition="'$(TargetFramework)' == 'net6.0-ios'">iossimulator-x64</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">maccatalyst-x64</RuntimeIdentifier>
+    <RuntimeIdentifiers Condition="'$(TargetFramework)' == 'net6.0-android'">android-arm64;android-x64</RuntimeIdentifiers>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <IsEmulator Condition="'$(IsEmulator)' == ''">true</IsEmulator>
+    <AndroidEnableProfiler>true</AndroidEnableProfiler>
+    <AndroidLinkResources>true</AndroidLinkResources>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AndroidEnvironment Condition="'$(IsEmulator)' == 'true'" Include="Android\environment.emulator.txt" />
+    <AndroidEnvironment Condition="'$(IsEmulator)' != 'true'" Include="Android\environment.device.txt" />
+  </ItemGroup>
```

Comment:
Swap out the envvars depending on the target. This is because they use different IP/ports.